docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md

```diff
@@ -1,379 +1,371 @@
 
     ```csharp
-    resultsTextBox.Clear();
-    SumPageSizes();
-    resultsTextBox.Text += "\r\nControl returned to startButton_Click.";
-    ```
-
-    The code calls the method that drives the application, `SumPageSizes`, and displays a message when control returns to `startButton_Click`.
+    _resultsTextBox.Clear();
 
-3. The code for the synchronous solution contains the following four methods:
+    SumPageSizes();
 
-    - `SumPageSizes`, which gets a list of webpage URLs from `SetUpURLList` and then calls `GetURLContents` and `DisplayResults` to process each URL.
+    _resultsTextBox.Text += $"\nControl returned to {nameof(OnStartButtonClick)}.";
+    ```
 
-    - `SetUpURLList`, which makes and returns a list of web addresses.
+    The code calls the method that drives the application, `SumPageSizes`, and displays a message when control returns to `OnStartButtonClick`.
 
-    - `GetURLContents`, which downloads the contents of each website and returns the contents as a byte array.
+1. The code for the synchronous solution contains the following members:
 
-    - `DisplayResults`, which displays  the number of bytes in the byte array for each URL.
+    - `_urlList`, a read-only field that represents a list of web addresses.
+    - `SumPageSizes`, which gets a list of webpage URLs from the `_urlList` field and then calls `ProcessUrl` and `DisplayResults` to process each URL.
+    - `ProcessUrl`, which downloads the contents of each URL and returns the contents as a byte array.
+    - `DisplayResults`, which displays the number of bytes in the byte array for each URL.
 
-    Copy the following four methods, and then paste them under the `startButton_Click` event handler in MainWindow.xaml.cs:
+    Copy the following code, and paste it above the `OnStartButtonClick` event handler in *MainWindow.xaml.cs*:
 
     ```csharp
-    private void SumPageSizes()
+    readonly IEnumerable<string> _urlList = new string[]
     {
-        // Make a list of web addresses.
-        List<string> urlList = SetUpURLList();
-
-        var total = 0;
-        foreach (var url in urlList)
-        {
-            // GetURLContents returns the contents of url as a byte array.
-            byte[] urlContents = GetURLContents(url);
+        "https://docs.microsoft.com",
+        "https://docs.microsoft.com/azure",
+        "https://docs.microsoft.com/powershell",
+        "https://docs.microsoft.com/dotnet",
+        "https://docs.microsoft.com/aspnet/core",
+        "https://docs.microsoft.com/windows"
+    };
+    ```
 
-            DisplayResults(url, urlContents);
+    Copy the following methods, and paste them below the `OnStartButtonClick` event handler in *MainWindow.xaml.cs*:
 
-            // Update the total.
-            total += urlContents.Length;
-        }
+    ```csharp
+    void SumPageSizes()
+    {
+        var stopwatch = new Stopwatch();
+        stopwatch.Start();
```

Comment:
nit: these can be combined into one line using the Stopwatch.StartNew static method.